{
    "checksum": "1a6ff05d73b584c85642498fb4e70d08c4fc6de1a009b66191faeb1b765b7c65",
    "file": "exceptions.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 2,
    "package": "azure-cli",
    "path": "azure-cli/2.18.0-2/src/azure-cli-testsdk/azure/cli/testsdk/exceptions.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/azure-cli/2.18.0-2/",
        "metric": {
            "size": 631860
        },
        "pts_link": "https://tracker.debian.org/pkg/azure-cli",
        "sloc": [
            [
                "python",
                272023
            ],
            [
                "sh",
                963
            ],
            [
                "makefile",
                305
            ],
            [
                "javascript",
                34
            ],
            [
                "cs",
                29
            ],
            [
                "xml",
                16
            ],
            [
                "cpp",
                12
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/azure-cli",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/azure-cli/2.18.0-2/src/azure-cli-testsdk/azure/cli/testsdk/exceptions.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 1294,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "2.18.0-2"
}